package.json

```diff
@@ -11,9 +11,11 @@
     "heroku-serve": "http-server",
     "heroku-postbuild": "npm run build --  --entry static-pages,style",
     "lint": "npm run lint:js && npm run lint:sass",
-    "lint:changed": "npm run lint:js:changed && npm run lint:sass",
+    "lint:changed": "npm run lint:js:changed && npm run lint:ts:changed && npm run lint:sass",
     "lint:js": "eslint --quiet --ext .js --ext .jsx .",
     "lint:js:changed": "LIST=`git diff-index --name-only HEAD | grep \"\\.js[x]\\{0,1\\}$\"`; if [ \"$LIST\" ]; then eslint --quiet $LIST; fi",
+    "lint:ts": "tslint -c tslint.json -p tsconfig.json",
+    "lint:ts:changed": "LIST=`git diff-index --name-only HEAD | grep \"\\.ts[x]\\{0,1\\}$\"`; if [ \"$LIST\" ]; then tslint -c tslint.json $LIST; fi",
```

Comment:
Having a separate file type means we need to run two linters, which maybe isn't great, but doesn't seem so bad. My biggest concern here is the potential differences between our eslint config and our tslint config.